extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -248,7 +295,7 @@ angular.module('oppia').component('literallyCanvasDiagramEditor', {
             defaultStrokeWidth: DEFAULT_STROKE_WIDTH,
```

Comment:
There is too much duplication between these lines and those below. Please find a way to reduce it.

Reply:
Done.